src/components/new-suggestion/areas/instruments/instruments.area.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react"
+import InstrumentsEdit from "./edit/instruments-edit"
+import InstrumentSearch from "@/components/search/instrument/instrument-search"
+import { Instrument } from "@/types/database-types"
+import { setActiveArea, updateNewSuggestion } from "@/redux/slices/new-suggestion.slice"
+import { AppDispatch, AppState } from "@/redux/store"
+import { useDispatch, useSelector } from "react-redux"
+import { NewSuggestionInstrument } from "@/interfaces/new-suggestion"
+import { Area } from "@/constants/area"
+
+interface InstrumentsAreaProps {
+  instruments: Instrument[]
+}
+
+const InstrumentsArea = ({ instruments }: InstrumentsAreaProps) => {
+  const dispatch: AppDispatch = useDispatch()
+  const newSuggestion = useSelector((state: AppState) => state.newSuggestion.suggestion)
+  const [canProceed, setCanProceed] = useState<boolean>(newSuggestion.instruments.length > 0)
+
+  useEffect(() => {
+    setCanProceed(newSuggestion.instruments.length > 0)
+  }, [newSuggestion])
+
+  const onInstrumentSelected = (instrument: Instrument): boolean => {
+    const newItem: NewSuggestionInstrument = {
+      instrument: instrument,
+      description: "",
+    }
+
+    dispatch(
+      updateNewSuggestion({
+        ...newSuggestion,
+        instruments: [...newSuggestion.instruments, newItem],
+      })
+    )
+    return true
+  }
+
+  const onSubmit = () => {
+    dispatch(setActiveArea(Area.Review))
```

Comment:
User can still go to review page by clicking on it in progress bar. Could you prevent this from happening? There is already a method that checks it for suggestions. I believe you can easily copy this to adjust it for your situation.